release-tools/metabuild.sh

```diff
@@ -52,6 +52,19 @@ add_makefile_prog() {
 	fi
 }
 
+get_cpusupport_cflags() {
+	src=$1
```

Comment:
You're not using `src` here.  Also, `src` is being set to the name of the `.o` file...